network-api/networkapi/wagtailpages/templates/wagtailpages/article_page.html

```diff
@@ -18,7 +18,7 @@
       <div class="offset-sm-1 col-sm-10 col-12">
         {% if page.show_side_share_buttons %}
           <div class="article-summary-share-links d-lg-block d-none {% if get_titles %}with-top-offset{% endif %}">
-            <div class="share-button-group-wrapper" data-version="mini" data-layout="stacked" data-share-text="{% blocktrans with title=page.title %}{{ title }} by @mozilla{% endblocktrans %}" data-link="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}"></div>
+            {% include "./fragments/share-buttons.html" with version="mini" layout="stacked" share_text=page.title|add:' by @mozilla' %}
```

Comment:
Will this cause problems for localization? @TheoChevalier

Reply:
Also if this can use either single or double quotes, let's stick with double quotes for consistency.

Reply:
Right, this doesn’t work — I have a follow up PR with a different approach waiting to be merged in this one https://github.com/mozilla/foundation.mozilla.org/pull/8261

Reply:
PR is merged! Going to mark this conversation as resolved